.make/docker-compose.integration-test.yaml

```diff
@@ -0,0 +1,14 @@
+version: '2.1'
+services:
+  postgres_integration_test:
+    image: centos/postgresql-95-centos7
```

Comment:
shouldn't it be `registry.centos.org/postgresql/postgresql:9.6` - this is at least used in other f8 services